fingerprint-api-simulator/README.md

```diff
@@ -0,0 +1,66 @@
+## Fingerprint Scanning Simulation API
+
+This CLI tool is designed to get you up and running with a mock fingerprint scanner that can be accessed via a request to your `localhost`.
+
+### Quickstart
+
+Install the package
+
+```
+npm i @kiva/fingerprint-api-simulator
+```
+
+Once installed, set up a script in your `package.json` file.
+
+```
+{
+    ...
+    "scripts": {
+        "simulator": "ssim"
```

Comment:
what does `ssim` stands for here?

Reply:
It's the exported CLI command from the package. Contraction for "SSIrius Simulator". https://github.com/kiva/ssi-wizard-sdk/pull/77/files#diff-bb2a0a17b6a3e93b5eaf107b25a79cf73ae186bc65759c6742ff756966deb308R19

It's not a good name, but I don't want the CLI command to be `npx runfingerprintapisimulator --file ...` because who wants to type all that?

Reply:
The reason I asked is because ssim is a term for image comparison and fingerprints are images that we compare. Reference: https://www.imatest.com/docs/ssim/#:~:text=The%20Structural%20Similarity%20Index%20(SSIM,image%20and%20a%20processed%20image.

Reply:
Oh, then it's a much worse name than I'd realized. Will think of something better.